app/models/datawarehouse_node.rb

```diff
@@ -0,0 +1,9 @@
+class DatawarehouseNode < ApplicationRecord
```

Comment:
I have a feeling this should be an ElasticsearchNode. We will not use this for hawkular (there is already a hawkular server)